cmd/nephe-controller/main.go

```diff
@@ -18,17 +18,18 @@ import (
 	"flag"
 	"os"
 
-	"k8s.io/apimachinery/pkg/runtime"
-	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
-	ctrl "sigs.k8s.io/controller-runtime"
-
 	antreanetworking "antrea.io/antrea/pkg/apis/controlplane/v1beta2"
 	antreatypes "antrea.io/antrea/pkg/apis/crd/v1alpha2"
 	crdv1alpha1 "antrea.io/nephe/apis/crd/v1alpha1"
 	runtimev1alpha1 "antrea.io/nephe/apis/runtime/v1alpha1"
 	"antrea.io/nephe/pkg/apiserver"
 	controllers "antrea.io/nephe/pkg/controllers/cloud"
 	"antrea.io/nephe/pkg/logging"
+
+	"k8s.io/apimachinery/pkg/runtime"
```

Comment:
system, third party and then antrea imports... move this up

Reply:
Done